lib/routes/user/home/wallet_dashboard.dart

```diff
@@ -18,58 +19,116 @@ class WalletDashboard extends StatelessWidget {
 
   WalletDashboard(this._accSettings, this._accountModel, this._height, this._offsetFactor, this._onCurrencyChange);
 
+  @override
+  State<StatefulWidget> createState() {
+    return new WalletDashboardState();
+  }
+}
+
+class WalletDashboardState extends State<WalletDashboard> {
+  static const CHART_MAX_VERTICAL_OFFSET = 70.0;
+  static const CHART_MAX_HORIZONTAL_OFFSET = 30.0;
+  static const BALANCE_OFFSET_TRANSITION = 30.0;
+  bool _showFiatCurrency = false;
+  bool _isPressed = false;
+
+  BreezBridge _breezLib;
+  double _convertedBalance = 0.0;
+
+  @override
+  void initState() {
+    super.initState();
+    _breezLib = new ServiceInjector().breezBridge;
+  }
+
+  void _getExchangeRates() async {
+    Rates _rate = await _breezLib.rate();
```

Comment:
It is better that the UI won't access directly the services layer (e.g breezLib), the exchange rates should come from the bloc.

Reply:
I also think we should add fiatBalance on the AccountModel as a calculated getter based on the btc balance and the conversion rate.